requirements.txt

```diff
@@ -1,23 +1,43 @@
 alabaster==0.7.10
```

Comment:
All these new requirements will probably add to the CircleCI build time, but it's probably not a thing I should be complaining about.

Reply:
I think they're transitive dependencies, although I'm not an experienced Pythonite.

Basically, I've run `pip freeze > requirements.txt` after installing sphinx-autobuild.

Reply:
https://nvie.com/posts/pin-your-packages/ suggests that I'm wrong and we should pin the dependencies.

Reply:
Anyway, I see that `pip freeze` has arguments. Maybe adding `--user` gets less stuff in.

Reply:
OK, `--user` gives nothing, and `--local` gives the same output. I don't know if we can try anything else.

It feels like it's OK, though.

Reply:
Nah. I wish I knew more Python! I'm sorry for all the confusion. With the new requirements it doesn't work right. Let me fix it